manim/scene/vector_space_scene.py

```diff
@@ -361,11 +348,7 @@ def position_y_coordinate(
         return y_coord
 
     def coords_to_vector(self, vector, coords_start=2 * RIGHT + 2 * UP, clean_up=True):
-        """
-        This method writes the vector as a column matrix (henceforth called the label),
-        takes the values in it one by one, and form the corresponding
-        lines that make up the x and y components of the vector. Then, an
-        Vector() based vector is created between the lines on the Screen.
+        """Write the vector as a column matrix (henceforth called the label), take the values in it one by one, and form the corresponding lines that make up the x and y components of the vector. Then, an Vector() based vector is created between the lines on the Screen.
```

Comment:
Needs a summary, any ideas?

Reply:
```suggestion
        """Animate the making of a vector from coords passed as a (2,1) ndarray
```
```suggestion
        """Animate the making of a vector from coords passed as a 2x1 column matrix
```
Anything could work.